modules/starboard/handleStarboard.js

```diff
@@ -0,0 +1,63 @@
+const { embedTypeV2, formatDiscordUserName } = require('../../src/functions/helpers');
+
+module.exports = async (client, msgReaction, isReactionRemove = false) => {
+    if (!client.botReadyAt) return;
+    const msg = msgReaction.message;
+    if (!msg.guild) return;
+    if (msg.guild.id !== client.guildID) return;
+    if (msgReaction.partial) msgReaction = await msgReaction.fetch();
+
+    const starConfig = client.configurations['starboard']['config'];
+    if (!starConfig || (starConfig.emoji !== msgReaction.emoji.name && starConfig.emoji !== msgReaction.emoji.id)) return;
+    if (isNaN(starConfig.minStars)) return disableModule('starboard', localize('starboard', 'invalid-minstars', {stars: starConfig.minStars}));
```

Comment:
`localize` is not imported in this file. Please add an import statement

Reply:
Neither is disableModule, as a matter of fact

Reply:
ESLint should be configured to flag that

Reply:
good point, feel fee to add that or I'll look into that